docs/docs/features/Formulas.md

```diff
@@ -22,3 +22,33 @@ The root object `db` has the following functions:
 
 - `js` : execute a javascript function that you previously defined in the `js` folder of your table in the database or plugin global settings. (I.E.: `db.js.myFunction( arg1, arg2)`)
 
+## Examples
```

Comment:
I like the idea to include examples

When can include sections. Your PR could be tittled as Datetime and includes the link to Luxon documentation

[luxon](https://moment.github.io/luxon)